models/intermediate/marketo__change_data_scd.sql

```diff
@@ -1,12 +1,14 @@
 {{
     config(
         materialized='incremental',
-        partition_by = {'field': 'valid_to', 'data_type': 'date'},
-        unique_key='lead_day_id'
+        partition_by = {'field': 'valid_to', 'data_type': 'date'} if target.type != 'spark' else ['valid_to'],
```

Comment:
Same note from the source, can we also add databricks to the target.type?
```
not in ('spark','databricks')
```

Reply:
As we chatted about earlier, for incremental models, I have updated `target.type != 'spark'` to  `target.type not in ['spark', 'databricks']`. For the source yml, I have left as-is.